frontend/src/features/mapSideBar/tabs/ltsa/LtsaOwnershipInformationTitleOwnerForm.tsx

```diff
@@ -0,0 +1,118 @@
+import { Input } from 'components/common/form';
+import { FieldArray, getIn, useFormikContext } from 'formik';
+import { LtsaOrders, TitleOwner } from 'interfaces/ltsaModels';
+import * as React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import styled from 'styled-components';
+import { withNameSpace } from 'utils/formUtils';
+
+import { SectionField, StyledFieldLabel } from '../SectionField';
+
+export interface ILtsaOwnershipInformationTitleOwnerFormProps {
+  nameSpace?: string;
+}
+
+export const LtsaOwnershipInformationTitleOwnerForm: React.FunctionComponent<ILtsaOwnershipInformationTitleOwnerFormProps> = ({
+  nameSpace,
+}) => {
+  const { values } = useFormikContext<LtsaOrders>();
+  const titleOwners = getIn(values, withNameSpace(nameSpace, 'titleOwners')) ?? [];
+
+  const getJoinedFieldValues = (input1?: string, input2?: string): string => {
```

Comment:
could this be summarized as return [input1, input2].join(', ') ? this seems a bit verbose.